python/google/protobuf/internal/well_known_types_test.py

```diff
@@ -372,6 +372,56 @@ def testInvalidDuration(self):
         r'Duration is not valid\: Sign mismatch.',
         message.ToJsonString)
 
+  def testTimestampOrdering(self):
+    a, b = timestamp_pb2.Timestamp(), timestamp_pb2.Timestamp()
+    a.seconds, a.micros = 0, 1
+    b.seconds, b.micros = 1, 0
```

Comment:
Add test for Timestamp that is not normalized and add comments to indicate the result may not correct if Timestamp is not normalized.

Reply:
I replaced `_comparable_vector(…)` with `_comparable_scalar(…)` which returns `ToNanoseconds()` for non-datetimes so will work with non-normalized values, and added tests